tests/foreman/cli/test_satellitesync.py

```diff
@@ -1214,52 +1300,84 @@ def test_negative_import_invalid_path(self, module_org):
             '--metadata-file option'
         ) in error.value.message
 
-    @pytest.mark.skip_if_open("BZ:1998626")
-    @pytest.mark.skip_if_open("BZ:2067275")
     @pytest.mark.tier2
     def test_negative_export_cv_with_on_demand_repo(
-        self, export_import_cleanup_function, module_org
+        self, export_import_cleanup_module, target_sat, module_org
     ):
-        """Exporting CV version having on_demand repo throws error
+        """Exporting CV version having on_demand repo throws error with "fail-on-missing-content" option set
 
         :id: f8b86d0e-e1a7-4e19-bb82-6de7d16c6676
 
         :steps:
 
-            1. Create product and on-demand repository with custom contents
-            2. Sync the repository
-            3. Create CV with above product and publish
-            4. Attempt to export CV version contents to a directory
+            1. Create product
+            2. Create repos with immediate and on_demand download policy
+            3. Sync the repositories
+            4. Create CV with above product and publish
+            5. Attempt to export CV version contents
 
         :expectedresults:
 
-            1. Export fails with error '"message": "NOTE: Unable to fully
-                export this version because it contains repositories without the
-                'immediate' download policy. Update the download policy and sync
-                affected repositories. Once synced, republish the content view and
-                export the generated version."'
+            1. Export fails with error and no export is created
+
+        :BZ: 1998626, 2067275
```

Comment:
@rmynar Could you confirm if these tests are expected for 6.11.z and run PRT for test results?

Reply:
yes, these changes are expected to be in 6.11.z